groovy/postInit/chemistry/ElectrolysisOverhaul.groovy

```diff
@@ -30,6 +30,17 @@ ELECTROLYZER.recipeBuilder()
         .duration(200)
         .buildAndRegister()
 
+ELECTROLYZER.recipeBuilder()
```

Comment:
this doesnt work irl, use the fluidized bed reactor recipe

Reply:
so just change machine to fluidized bed reactor?

Reply:
and also can i know where you get the chemistry knowledge from? itd be easier to make realistic recipes if i knew.

Reply:
there is a recipe in game to recycle hydrogen chloride in the fluidized bed reactor, just remove this recipe

Reply:
as for the knowledge, I recommend the following books:

Atkins' Chemical Principles
Klein's Organic Chemistry

for more advanced knowledge:

Meissner's Inorganic Chemistry
Atkins & Shrivers' Inorganic Chemistry
Clayden's Organic Chemistry
Atkins' Physical Chemistry
Introduction to Semimicro Qualitative Analysis

for industrial chemistry knowledge (most applicable to susy)

Ullmann's Encyclopedia of Industrial Chemistry
Kirk-Othmer's Concise Encyclopedia of Industrial Chemistry

for very niche knowledge and/or more detailed specifications, look for papers and patents

Reply:
You will need a working knowledge of calculus 3 for physical chemistry, just fyi.

Reply:
okay thank you

Reply:
are these book free to read on the internet?